doc/simulators/simulators.rst

```diff
@@ -0,0 +1,14 @@
+.. _simulators:
+
+#################
+Simulators
+#################
+
+This page hosts a list of supported simulators with ros2_control implementations.
+To add your ros2_control integration, submit a PR to this page on Github!
+
+* `Gazebo Classic <https://github.com/ros-controls/gazebo_ros2_control>`__
+* `Gazebo <https://github.com/ros-controls/gazebo_ros2_control>`__
```

Comment:
```suggestion
* `Gazebo <https://github.com/ros-controls/gazebo_ros2_control>`__
* `Gazebo Classic <https://github.com/ros-controls/gazebo_ros2_control>`__
```

Nit: not that this is an ordered list but put Gazebo Classic second as it will eventually be retired and we should drive users towards Gazebo